views/partials/affected-emails.hbs

```diff
@@ -0,0 +1,21 @@
+ <div class="resolve-breaches-wrapper drop-shadow">
+  <h3 class="bold resolve-headline txt-cntr">{{ resolutionStrings.subhead }}</h3>
+  <div class="affected-emails-list-wrapper">
+    <p class="resolution-message">{{{ resolutionStrings.message }}}</p>
+    <ul class="affected-emails-list">
+      {{#each affectedEmails}}
+        <li class="flx space-between affected-email-list-item">
+          <p class="flx cntr demi affected-email">{{ emailAddress }}</p>
+            <button data-affected-email="{{ emailAddress}}" data-passwords-exposed="{{ ./../breachExposedPasswords }}" data-recency-index="{{ recencyIndex }}" data-is-resolved="{{ isResolved }}"
```

Comment:
@lesleyjanenorton I'm changing this to `data-is-already-resolved` to make the back-end logic more straight-forward.

Reply:
oops, nvm.